.azure-pipelines/pipelines.yml

```diff
@@ -62,6 +62,9 @@ stages:
 # Presubmit/default
 - ${{ if eq(variables.pipelineDefault, true) }}:
   - template: stages.yml
+    parameters:
+      checkStageDeps:
+      - env
```

Comment:
this is not a clean revert, context for this change?

Reply:
speeding up the test run to get a result quicker

good catch - ill remove